MyStore/src/include/InfoProduct.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <vector>
+#include <string>
+
+struct InfoProduct
+{
+	std::string m_product_name = {};
+	std::string m_product_color = {};
+};
```

Comment:
Somehow this file is included incorrectly into solution. Try to remove it in Visual Studio and re-add again.